conf/alluxio-site-properties.dora-local-minimum.template

```diff
@@ -0,0 +1,19 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+# This template contains a set of minimum configurations to run a dora cluster locally.
+
```

Comment:
LGMT. Just one small suggestion: The default value of `alluxio.worker.page.store.sizes` is `epoll`, but MacOS doesn't support `epoll`, so MacOS users need to set `alluxio.worker.page.store.sizes=nio` to run DORA. Would it be better to add `alluxio.worker.page.store.sizes=nio` to `alluxio-site-properties.dora-local-minimum.template`?

Reply:
it's defsaulted to nio already so no need to change